### DISCUSSION TITLE
INSUFFICIENT_FUNDS upon contract deployment

### DISCUSSION THREAD

**Discussion by z1cb4** [yuichiroaoki/poly-flash#35] [Q&A]
Hi,

i am receiving an INSUFFICIENT_FUNDS error up the smart contract deployment.
.env File has been created , private key starting with 0x has been added , and
the account is funded with 10 Matic & some WETH.
What could be the reason? 

thnx

**Comment by yuichiroaoki** [comment]
That's weird. Does it work on the mainnet fork?

**Comment by AndrewHei** [comment]
I also have this problem. I have about 10 MATIC, but it said insufficient funds.
yarn test was successful so not sure what is the problem 🤔

**Comment by bloodha** [comment]
hello i have a question how can i execute the smart contrat?
